conf/messages.en

```diff
@@ -737,7 +737,7 @@ fileManagement.noFiles.paragraph1 = You do not have any TGP record files ready f
 fileManagement.availableFiles.table.caption = TGP records files
 fileManagement.availableFiles.table.header1 = Date and time requested
 fileManagement.availableFiles.table.header2 = Expiry date
-fileManagement.availableFiles.table.header3 = File
+fileManagement.availableFiles.table.header3 = File(.csv)
```

Comment:
I think there might need to be a space between the bracket and File "File (.csv)"